db/migrate /20231024092305_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter, default: 0
+      t.integer :likes_counter, default: 0
+      t.timestamps
+      t.references :user, foreign_key: true
```

Comment:
- [ ] In keeping with the ERD diagram shown below, instead of a `user_id` column, we need an `author_id` column. You can achieve this like this: 👇 
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```
![image](https://github.com/MenendezON/Blog-app/assets/46839548/e53ddaa1-e2db-4c2c-babd-464d7f018c44)